src/__tests__/cypress/integration/channels.spec.js

```diff
@@ -0,0 +1,56 @@
+describe('Channels', () => {
```

Comment:
The whole `__tests__` folder should be moved up to the top-level.

Reply:
This is short syntax for a React Fragment and is necessary to return multiple elements without a parent

Reply:
@faustbrian The project is bootstrapped with Create React App and requires the `__tests__` folder to be placed inside the `src` folder. If you move the folder to the root it will create all kinds of problems and would need quite some extra work to get it working properly. Worst case scenario an eject is necessary, and this would not work well with the goals set out for this project.